components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabIntentProcessor.kt

```diff
@@ -34,7 +35,7 @@ class CustomTabIntentProcessor(
         val safeIntent = SafeIntent(intent)
         val url = safeIntent.dataString
 
-        return if (!url.isNullOrEmpty() && matches(intent)) {
+        return if (!url.isNullOrEmpty() && matches(intent) && !shouldAlwaysUseBrowserUI(intent)) {
```

Comment:
How does a user actually opt-out of custom tabs? Is that up to the app? I'm confused why the app wouldn't just send a regular VIEW intent in that case? 🤔

Reply:
I agree, this is a strange flag to have. Are they trying to provide the equivalent of 'Open in browser' as part of the config? Is it supposed to be a user-option when the calling app doesn't let you switch out of a custom tab? Had to tell what the intention is.

I don't think we need to support this since all (most) of our browsers support the functionality that this provides, but maybe that should be up to the browser app as well. 🤷‍♂